react/MonthPicker/CustomMonthPicker/CustomMonthPicker.js

```diff
@@ -4,24 +4,10 @@ import React, { Component } from 'react';
 import PropTypes from 'prop-types';
 
 import classnames from 'classnames';
-import range from 'lodash/range';
 
+import getYearOptions from './getYearOptions';
 import Dropdown from '../../Dropdown/Dropdown';
 
-const getYearOptions = () => {
-  const maxYear = new Date().getFullYear();
```

Comment:
I cant see where this behaviour is being retained. Should `maxYear` not be a default prop of this year?

Reply:
Max year defaults to the current year. Are you saying it shouldn't?

Reply:
I agree it should, but since removing this line I cant see where the defaulting is done

Reply:
Check out the public `MonthPicker` component. https://github.com/seek-oss/seek-style-guide/pull/368/files#diff-36c8b55b5646996302a1b706cd547849R49

Reply:
Ignore me, my PR view was missing files for some reason 🤔